hnn_core/viz.py

```diff
@@ -62,8 +66,159 @@ def plt_show(show=True, fig=None, **kwargs):
         (fig or plt).show(**kwargs)
 
 
+def plot_extracellular(times, data, sfreq, window_len=None, tmin=None,
+                       tmax=None, ax=None, decim=None, color=None,
+                       voltage_offset=None, voltage_scalebar=None,
+                       contact_labels=None, show=True):
+    """Plot extracellular electrode array voltage time series.
+
+    Parameters
+    ----------
+    times : list | Numpy array
+        Sampling times (in ms).
+    data : Two-dimensional Numpy array
+        The extracellular voltages as an (n_contacts, n_times) array.
+    sfreq : float
+        Data sampling rate (in Hz).
+    window_len : float
+        If set, apply a Hamming-windowed convolution kernel of specified length
+        (in ms) to the data before plotting. Default: None
```

Comment:
this is a bit inconsistent with the API of Dipole object where there is a `smooth` method.

Reply:
I though of that too, will make a proposal.

Reply:
see 8c934a3